test/cli-test.js

```diff
@@ -100,6 +100,21 @@ describe('command line interface', function () {
     });
   });
 
+  it('should create swagger.json by default when the API input is from definition file', function (done) {
+    var goodInput = process.env.PWD + '/bin/swagger-jsdoc.js -d test/fixtures/api_definition.js example/routes.js';
```

Comment:
I think you mean `"/bin/swagger-jsdoc.js -d test/fixtures/api_definition.js"` :)